tapir.json

```
@@ -0,0 +1,15 @@
+{
+    "80002": [
+        "https://polygon-amoy-bor-rpc.publicnode.com",
+        "https://rpc-amoy.polygon.technology"
+    ],
```

Let's add a public drpc endpoint in here as well (same suggestion in other files):

```suggestion
        "https://rpc-amoy.polygon.technology",
        "https://polygon.drpc.org"
    ],
```